samples/drivers/display/src/main.c

```diff
@@ -34,6 +34,10 @@ LOG_MODULE_REGISTER(sample, LOG_LEVEL_INF);
 #define DISPLAY_DEV_NAME DT_LABEL(DT_INST(0, fsl_imx6sx_lcdif))
 #endif
 
+#if DT_NODE_HAS_STATUS(DT_INST(0, solomon_ssd16xxfb), okay)
```

Comment:
What do you want to describe in the commit message?

The correct fix would be:
```diff c
diff --git a/samples/drivers/display/src/main.c b/samples/drivers/display/src/main.c
index d6cc95c96c..611402d971 100644
--- a/samples/drivers/display/src/main.c
+++ b/samples/drivers/display/src/main.c
@@ -22,8 +22,8 @@ LOG_MODULE_REGISTER(sample, LOG_LEVEL_INF);
 #define DISPLAY_DEV_NAME DT_LABEL(DT_INST(0, solomon_ssd1306fb))
 #endif
 
-#if DT_NODE_HAS_STATUS(DT_INST(0, gooddisplay_gdeh0213b1), okay)
-#define DISPLAY_DEV_NAME DT_LABEL(DT_INST(0, gooddisplay_gdeh0213b1))
+#if DT_NODE_HAS_STATUS(DT_INST(0, solomon_ssd16xxfb), okay)
+#define DISPLAY_DEV_NAME DT_LABEL(DT_INST(0, solomon_ssd16xxfb))
 #endif
 
 #if DT_NODE_HAS_STATUS(DT_INST(0, sitronix_st7789v), okay)
```

Reply:
I agree. It can just as well be generic support for all ssd16xx based displays. Fixed